cyber-resilience-sig/whitepapers/stewards-and-cra.md

```diff
@@ -13,23 +13,415 @@ License: CC-BY 4.0
+
+The Steward may have the policy in paper or electronic form, but it must be "verifiable". As the relationships between
+Stewards and their Projects vary, the Policy should take into account their common mode of operations.
+
+Market Surveillance Authorities may request such a Security Policy.
+
+References: Article 24(1)
+
+        Cyber Resilience Act, Article 24(1):
+        
+        Open-source software stewards shall put in place and document in a verifiable manner a cybersecurity policy to foster 
+        the development of a secure product with digital elements as well as an effective handling of vulnerabilities by the 
+        developers of that product. That policy shall also foster the voluntary reporting of vulnerabilities as laid down in
+        Article 15 by the developers of that product and take into account the specific nature of the open-source software 
+        steward and the legal and organisational arrangements to which it is subject. That policy shall, in particular, include
+        aspects related to documenting, addressing and remediating vulnerabilities and promote the sharing of information
+        concerning discovered vulnerabilities within the open-source community.
+
+In Open Source Projects, Security Policies are public (then the publication requirement is fulfilled) and published on
+the Project site and/or included in their source code repository.
+
+In practice, we see two patterns related to security policies:
+- Either the supporting organization publishes one, and Projects link to it in their documentation. This may include some minor
+clarifications, for example a specific link or email address to report vulnerabilities in that specific Project.
+- Or, Projects publish their policies based on the template provided by the supporting organization.
+
+In both cases, both the Steward and the Project will need to check if the Policy they provide includes all needed elements and that it
+is followed. In the case the Project does not follow the Policy, the Steward needs to define the way to follow, see the [related FAQ entry](https://github.com/orcwg/cra-hub/issues/292).
+
+As the Policy needs to be followed, the Steward and the Project will need to agree on a way to create the policy and provide changes.
+For example, accepting a generic Policy of a Foundation-Steward may be a requirement for creation of a project or accepting that project under that Foundation.
+
+What Stewards and Project will need:
+- The published Policy (that is larger in scope than a typical current security policy)
+- Evidence that the Policy is followed
+
+What resources could be created to help fulfilling that requirement:
+- A description about how to demonstrate that the policy is put in place - how to record it and store it such that an independent
+party can check that the policy exists (version-controlled documents, maintaining a change log, ...)
```

Comment:
…Or just look for a file named `SECURITY.md` in the project repository directory? This is what we urge CPAN package maintainers to do: https://security.metacpan.org/2025/01/05/add-a-security-policy.html

Reply:
Addressed in https://github.com/orcwg/orcwg/pull/209
Please have a look